static/components.interfaces.ts

```diff
@@ -22,9 +22,8 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import {CompilerFilters} from '../types/features/filters.interfaces';
+import {CompilerOutputOptions, ParseFiltersAndOutputOptions} from '../types/features/filters.interfaces';
```

Comment:
I can't see that `ParseFiltersAndOutputOptions` is used in this file

Reply:
Yes sorry, went for some solution then changed it back to something else. I was blindly expecting the linter to rejects useless imports. Will fix them now, thanks for pointing them out!